onscale_client/client.py

```diff
@@ -1661,6 +1679,27 @@ def linked_file_check(file, linked_files):
                 )
                 material_file_ids.append(mat_blob_id)
 
+        # Wait for meshing to finish
+        wait_for_blob(blob_type="MESHAUTO",
+                      object_id=job.design_instance_id,
+                      object_type="DESIGNINSTANCE")
```

Comment:
shouldn't you include the `timeout_secs` argument so that there is no danger of an infinite loop?

Reply:
will do